src/Serializer/JsonLd/RestaurantMenuNormalizer.php

```diff
@@ -169,6 +193,15 @@ public function normalize($object, $format = null, array $context = array())
                         }, $allergens));
                     }
 
+                    $image_1x1 = $this->getProductImagePath($product, '1:1');
+                    if ( $image_1x1 ) {
+                        $item['image_1x1'] = $image_1x1;
+                    }
+                    $image_16x9 = $this->getProductImagePath($product, '16:9');
+                    if ( $image_16x9 ) {
+                        $item['image_16x9'] = $image_16x9;
```

Comment:
IMHO all images should be under an `images` property, in an array

```json
{
  "images": [
    {
      "ratio": "16:9",
      "url": "https://example.com/image.jpg"
    }
  ]
}
```

Reply:
I agree, nice improvement :)